src/exporter/DaemonMetricCollector.cc

```diff
@@ -116,72 +116,83 @@ void DaemonMetricCollector::dump_asok_metrics() {
       failures++;
       continue;
     }
-
-    json_object counter_dump = boost::json::parse(counter_dump_response).as_object();
-    json_object counter_schema = boost::json::parse(counter_schema_response).as_object();
-
-    for (auto &perf_group_item : counter_schema) {
-      std::string perf_group = {perf_group_item.key().begin(),
-                                perf_group_item.key().end()};
-      json_array perf_group_schema_array = perf_group_item.value().as_array();
-      json_array perf_group_dump_array = counter_dump[perf_group].as_array();
-      for (auto schema_itr = perf_group_schema_array.begin(),
-                dump_itr = perf_group_dump_array.begin();
-           schema_itr != perf_group_schema_array.end() &&
-           dump_itr != perf_group_dump_array.end();
-           ++schema_itr, ++dump_itr) {
-        auto counters = schema_itr->at("counters").as_object();
-        auto counters_labels = schema_itr->at("labels").as_object();
-        auto counters_values = dump_itr->at("counters").as_object();
-        labels_t labels;
-
-        for (auto &label: counters_labels) {
-          std::string label_key = {label.key().begin(), label.key().end()};
-          labels[label_key] = quote(label.value().as_string().c_str());
-        }
-        for (auto &counter : counters) {
-          json_object counter_group = counter.value().as_object();
-          if (counter_group["priority"].as_int64() < prio_limit) {
-            continue;
+  try {
```

Comment:
wrong indent. and large code churn. please split this change into two.

Reply:
i've seperated the indentation into another commit

Reply:
i cannot find "another commit". this commit is difficult to review. as i cannot identify what's changed and what is not without comparing the code line by line. anyway, i don't want to block this PR. so i withdraw my "request to change".

Reply:
i had another commit, but squashed it considering i had refactored the function. apologies